shared/js/devtools/panel.js

```diff
@@ -125,6 +125,18 @@ const actionHandlers = {
             table.appendChild(row)
         }
     },
+    error: (m) => {
```

Comment:
See my comment from CSS about possibly renaming this to be `jsException`, but again fine either way.

Reply:
Done